lms/templates/ccx/schedule.html

```diff
@@ -159,8 +81,8 @@ <h2 id="ccx_schedule_unit">${_('Schedule a Unit')}</h2>
        var date = $(this).datepicker( "getDate" );
        if (date) {
            $(this).val(date.getFullYear() + "-" +
-                   ('0' + (date.getMonth() + 1)).slice(-2) + "-" +
-                   ('0' + date.getDate()).slice(-2));
+             ('0' + (date.getMonth() + 1)).slice(-2) + "-" +
```

Comment:
This is a copy of the date formatting code I commented on above. It would be good to simplify this. If it can't be simplified, please at least create a shared utility method.